code/CLI/CLIEditor.py

```diff
@@ -97,22 +97,182 @@ def do_list_classes(self, args):
 
     def do_create_field(self, args):
         executeCMD(self.model, self.command_history, "create_field", args.split())
+
+    def complete_create_field(self, text, line, begidx, endidx):
+        classes = fetch_classes(self.model)
+        # split the command into a list of words
+        args = line.split()
```

Comment:
You could probably make this cleaner if you calculate the argument's index. I mentioned that in the discord which you can do with 
```python
argIndex = (len(line.split()) - (0 if text == "" else 1))
```
This can help with the bugs where pressing tab right after completing the class name then replaces the class name with a visibility. 

Note: the above is a 0-based index so instead of `if len(args) == 1` it would be `if argIndex == 0`